packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1582,11 +1582,11 @@ describe('postgresql introspection stopgaps', () => {
     })
   }
 
-  describe('postgres partitioned', () => {
+  describe('warning 27', () => {
```

Comment:
I think worthwhile to keep the "topic" if the warning code here as well.
```suggestion
  describe('warning 27 - postgres partition tables', () => {
```